Sources/_StringProcessing/MatchingOptions.swift

```diff
@@ -150,6 +160,9 @@ extension MatchingOptions {
     case unicodeScalarSemantics
     case byteSemantics
     
+    // Swift-only default possessive quantifier
+    case possessiveByDefault
```

Comment:
Does this need to be modeled in this way? It seems like options would just have a quantification kind in it.

Reply:
This is a bitset internally, so it's modeling the tripartite state with two Booleans and an invariant check.